### PR TITLE
dns_amp rely on ra which is not set when additionnal RRs received

### PR DESCRIPTION
**Pull request by guikcd**
Found when using QUERYTYPE=NS and DOMAINNAME=. :)

### CODE REVIEW

**Comment by guikcd** [comment]
My usecase is interesting because i've just found that it does not report amplification possible with old bind servers. I'll update this PR.

**Comment by guikcd** [comment]
Old or misconfigured bind servers doesn't set ra flag but answer additional RRs (even with recursion disabled but additional-from-cache set to yes) which can lead to 9x amplification factor.

I have the possibility here not rely on ra flag or rely on number of answers / additional RRs (maybe better, i've started to implement it if needed). Feel free to discuss the implementation.

More information on: https://www.dns-oarc.net/oarc/articles/upward-referrals-considered-harmful

File: modules/auxiliary/scanner/dns/dns_amp.rb [hunk 1]
```diff
@@ -86,7 +86,7 @@ def scanner_prescan(batch)
     print_status("Sending DNS probes to #{batch[0]}->#{batch[-1]} (#{batch.length} hosts)")
     # Standard packet is 60 bytes. Add the domain size to this
     sendpacketsize = 60 + datastore['DOMAINNAME'].length
-    print_status("Sending #{sendpacketsize} bytes to each host using the IN ANY #{datastore['DOMAINNAME']} request")
```
Comment (wvu-r7) [inline modules/auxiliary/scanner/dns/dns_amp.rb]: Surprised this was hardcoded to begin with.

**Comment by wvu-r7** [comment]
Thanks! Working here!

**Pull request merged**